lib/iex/test/iex/autocomplete_test.exs

```diff
@@ -408,4 +408,50 @@ defmodule IEx.AutocompleteTest do
     assert expand('take(') == {:yes, '', ['take(enumerable, amount)']}
     assert expand('derive(') == {:yes, '', ['derive(protocol, module, options \\\\ [])']}
   end
+
+  @tag :tmp_dir
+  test "path completion inside strings", %{tmp_dir: dir} do
+    File.cd!(dir)
+
+    try do
+      File.touch("./single1")
+      File.touch("./file1")
+      File.touch("./file2")
+      File.mkdir("./dir")
+      File.touch("./dir/file3")
+      File.touch("./dir/file4")
+
+      assert expand('".') == {:yes, '/', ['file2', 'single1', 'dir', 'file1']}
+      assert expand('"./sin') == {:yes, 'gle1"', []}
+      assert expand('"./fi') == {:yes, 'le', ['file2', 'file1']}
+      assert expand('"./d') == {:yes, 'ir/', []}
+      assert expand('"./dir/') == {:yes, 'file', ['file3', 'file4']}
+      assert expand('"./dir/#\{Str') == {:yes, '', ['Stream', 'String', 'StringIO']}
+
+      {:yes, [], list} = expand('"/')
+      assert is_list(list)
+
+      case :os.type() do
+        {:win32, _} ->
+          assert 'Users' in list
+          assert 'Windows' in list
+
+        _ ->
+          assert 'bin' in list
+          assert 'etc' in list
+          assert 'opt' in list
+          assert 'var' in list
+      end
+
+      default_expand = expand('{')
+      assert expand('"./dir/#\{') == default_expand
+    after
+      File.rm("./single1")
+      File.rm("./file1")
+      File.rm("./file2")
+      File.rm("./dir/file3")
+      File.rm("./dir/file4")
+      File.rm("./dir")
```

Comment:
I don't think you need to remove them (nor the try/catch) if using the tmp_dir feature?

Reply:
You are totally right, we don't need to remove them as it clears the tmp dir before starting the tests. Should I include `tmp` dir on `.gitignore` then? Just to make sure no one commits that?

Reply:
Don’t we have a tmp directory for ExUnit yet? Or we do but it is in a separate location than the one the tag creates it?

Reply:
We already have this:

https://github.com/elixir-lang/elixir/blob/master/.gitignore#L4

can we configure it to use it instead?

Reply:
Actually, nevermind, it did create the `tmp` folder on the root cause I ran the test only for that particular file from the actual root, so everything is fine